### ISSUE TITLE
how to add this MosaicOS function in Detectron2

### ISSUE THREAD

**Issue by hattergjj** [czhang0528/MosaicOS#2]
Hi
i saw in the README.md, this is a desctiption of Pre-trained models.  in this part, it says,"our implementations is based  on Detectron2". does this mean, this in a new Mosaic function has added in Detectron2 ? or just a off-line data augmentation used by mosaic, then using into detectron2.
thanks in advance , if someone  anwsers

**Comment by czhang0528** [comment]
Hello, the new Mosaic is separated from Detectron2. The current script in this repo is done offline. You have to prepare the augmented data and annotations (i.e., bounding boxes) similar to the original COCO images.

You may also directly plug the current script into the dataloader of Detectron2 to make it online --- each time sampling N (e.g., 4) images and mosaicking --- which gives similar performance but slows down the training.

**Comment by hattergjj** [comment]
@czhang0528 Thank you for you anwers. 
when i try to do this Mosaic function in my data. it shows an emply list "segmentation":[] in output annotations.json file. i am a bit confused. Does this not allow mosaic function using in instance segmentation? but there are descriptions of pre-training model for instance segmentation in the README.md

**Comment by czhang0528** [comment]
I see your point. MosaicOS in this work only considers bounding box annotations. That is, we leave "segmentation" empty when preparing the annotations. No mask (e.g., polygon or RLE for COCO) conversion script is provided in the repo.

We mosaic object-centric images and use them (_only with box labels_) to improve instance segmentation. Please be referred to Section 6.4 of the main paper and Section G.2 of the supplementary material.

**Comment by hattergjj** [comment]
@czhang0528 thanks a lot！ and thank you for your team have done this awsome work!
